pkg/controller/restore_controller.go

```diff
@@ -283,6 +290,8 @@ func (r *restoreReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 	if err = kubeutil.PatchResource(original, restore, r.kbClient); err != nil {
 		log.WithError(errors.WithStack(err)).Info("Error updating restore's final status")
+		// delete from tracker so next reconcile fails the restore
```

Comment:
nit: s/delete from/add to/ in the comment

Reply:
```suggestion
		// add to failureTracker so next reconcile fails the restore
```